main.py

```diff
@@ -1,4 +1,9 @@
 from stix_shifter.scripts.stix_shifter import main
+import sys
 
+
+arguments = sys.argv
+for i in range(len(arguments)-1):
+    arguments[i-1] = arguments[i-1].replace("\\","")
```

Comment:
This is just a convenience since launch.json for VSCpode and the equivalent command line settings for PyCharm require escaping quotes and then the connector does not work

Reply:
ohk. I think you should remove this and just make a local change for your workspace.